File: scripts/probe_index_layout.py
#!/usr/bin/env python3
import pymysql
import os
conn = pymysql.connect(host='127.0.0.1', user='root', password=os.environ.get('RMX_DB_PASSWORD', 'm2m1234!'), database='rmx_db', charset='utf8mb4')
cur = conn.cursor()
cur.execute("SELECT mid, layout_srl, mlayout_srl, use_mobile, module, LEFT(content,80), LEFT(mcontent,80) FROM rx_modules WHERE mid='index'")
print(cur.fetchone())
cur.execute("SELECT layout_srl, title, path FROM rx_layouts")
for r in cur.fetchall():
    print(r)
conn.close()

File: scripts/probe_index_layout.php
<?php
define('__RX_BASEDIR__', dirname(__DIR__) . '/');
require __RX_BASEDIR__ . 'common/autoload.php';
Context::init();
$db = Rhymix\Framework\DB::getInstance();
$rows = $db->query("SELECT mid, layout_srl, mlayout_srl, use_mobile, module FROM rx_modules WHERE mid='index'")->fetchAll();
print_r($rows);
$rows2 = $db->query("SELECT layout_srl, title, path FROM rx_layouts")->fetchAll();
print_r($rows2);
